theme/ItaliaTheme/Views/_common.scss

```diff
@@ -193,3 +193,29 @@ div.sticky-wrapper {
 .slider-container {
   margin-bottom: 1.5rem;
 }
+
+.card-wrapper {
+  .custom-calendar-card {
+    .card-calendar {
+      text-align: left;
+      padding: 10px 0;
+      width: 116px;
+
+      .date-label {
+        width: 26px;
+        font-size: 1rem;
+        text-align: center;
+        margin-right: 5px;
+        font-family: roboto-light;
```

Comment:
per questo dovrebbe esserci la variabile. Roboto è installato nel tema? Non mi ricordo se è un font di agid, se tu lo vedi bene forse ce l'hai installato sul pc.

Reply:
Roboto è già presente nel tema in theme.js c'è import 'typeface-roboto-mono';

Reply:
ok, per questo non c'è nessuna variabile definita perchè roboto-light bootstrap italia non l'ha mai usato.
Possiamo lasciarlo cosi, oppure fare una variabile in _site_variables.scss del tipo $font-family-monospace-light, così la possono riusare tutti e se un giorno per un qualche motivo (anche nei siti figli) si vuole cambiare quel font si cambia solo la variabile

Reply:
aggiunta variabile